adjust VSCode configuration docs (#913)
apollographql/docs: src/content/basics/devtools/editor-plugins.mdx

To get all the benefits of the VS Code experience, it's best to link the schema that is being developed against before installing the extension. The best way to do that is by [publishing a schema](/graphos/delivery/publishing-schemas/) to the Apollo schema registry. After that's done:

1. Create an `apollo.config.js` file at the root of the project.

>    Alternatively, you can also create a `cjs`, `mjs`, or `ts` file with the same configuration.

Reviewer: ```suggestion
   Alternatively, you can create a `cjs`, `mjs`, or `ts` file with the same configuration.
```

};
```

Linking to the local schema won't provide all features, such as switching graph variants and performance metrics.

### Local schema files

You might not always have a running server to link to, so the extension also supports linking to a local schema file.
This is useful for working on a schema in isolation or for testing out new features.
To link to a local schema file, add the following to the `apollo.config.js` file:

```js
module.exports = {
  client: {
    service: {
      // can be a string pointing to a single file or an array of strings
      localSchemaFile: './path/to/schema.graphql'
    }
  }
};
```

### Client-only schemas

> One of the best features of the VS Code extension is the automatic merging of remote schemas and local ones when using integrated state management with Apollo Client. This happens automatically whenever schema definitions are found within a client project. By default, the VS Code extension will look for all JavaScript files under `./src` to find both the operations and schema definitions for building a complete schema for the application.

Author: ```suggestion
One of the best features of the VS Code extension is the automatic merging of remote schemas and local ones when using integrated state management with Apollo Client. This happens automatically whenever schema definitions are found within a client project. By default, the VS Code extension will look for all JavaScript, TypeScript and GraphQL files under `./src` to find both the operations and schema definitions for building a complete schema for the application.
```

  alt="Clicking the status bar icon to open the output pane"
/>

> If problems persist or the error messages are unhelpful, an [issue](https://github.com/apollographql/vscode-graphql/issues) can be opened on the `vscode-graphql` repository.

Reviewer: ```suggestion
If problems persist or the error messages are unhelpful, open an [issue](https://github.com/apollographql/vscode-graphql/issues) in the `vscode-graphql` repository.
```

> ## Other configuration options

Reviewer: This applies only to the Apollo Config file, right? If so, let's either 1) explicitly state that or move it under the ### Setting up an Apollo config option.

Author: Yes, but the same goes for "Local schemas", "Local schema files" and "Client-only schemas", which are weirdly not under "Setting up an Apollo config", but under "Getting Started", so I'm honestly a bit unsure on the whole document structure

Reviewer: Hah, got it. Noted that this entire thing could use a restructure.

### `client.tagName`

_Optional_ - custom tagged template literal.

> When using GraphQL with JavaScript or TypeScript projects, it is common to use the `gql` tagged template literal to write out operations. Apollo tools will be looking through your files for the `gql` tag to extract your queries, so if you use a different template literal, you can configure it like so:

Reviewer: ```suggestion
When using GraphQL with JavaScript or TypeScript projects, it is common to use the `gql` tagged template literal to write out operations. Apollo tools look through your files for the `gql` tag to extract your queries, so if you use a different template literal, you can configure it like so:
```
